src/security/initializable.cairo

```diff
@@ -1,7 +1,12 @@
-// SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts for Cairo v0.7.0 (security/initializable.cairo)
-
-#[starknet::contract]
+/// SPDX-License-Identifier: MIT
+/// OpenZeppelin Contracts for Cairo v0.7.0 (security/initializable.cairo)
+///
+/// # Initializable Component
+///
+/// The Initializable component provides a simple mechanism that executes
+/// logic once and only once. This can be useful for setting a contract's
+/// initial state in scenarios where a constructor cannot be used.
```

Comment:
```suggestion
//! SPDX-License-Identifier: MIT
//! OpenZeppelin Contracts for Cairo v0.7.0 (security/initializable.cairo)
//!
//! # Initializable Component
//!
//! The Initializable component provides a simple mechanism that executes
//! logic once and only once. This can be useful for setting a contract's
//! initial state in scenarios where a constructor cannot be used.
```